src/base/error_code.h

```diff
@@ -1739,7 +1739,9 @@
 #define ER_SP_SERVER_CRASHED                        -1359
 #define ER_SP_COMPILE_ERROR                         -1360
 
-#define ER_LAST_ERROR                               -1361
+#define ER_DISK_REMARKS_LENGTH_TOO_LONG             -1361
```

Comment:
```suggestion
#define ER_DISK_REMARKS_LENGTH_IS_TOO_LONG             -1361
```
ER_BO_FULL_DATABASE_NAME_IS_TOO_LONG와 일관성 있게

Reply:
넵! 수정했습니다.